packages/sdk/src/Portfolio/AssetManagers/MorphoBlueFlashLoanAssetManager.ts

```diff
@@ -0,0 +1,77 @@
+import { IDispatcherOwnedBeaconFactory, IMorphoBlueFlashLoanAssetManager } from "@enzymefinance/abis";
+import { type Address, type Client, type Hex, encodeFunctionData } from "viem";
+import { readContract } from "viem/actions";
+import { Viem } from "../../Utils.js";
+
+//--------------------------------------------------------------------------------------------
+// FACTORY
+//--------------------------------------------------------------------------------------------
+
+export function encodeDeployProxyConstructData(args: {
+  owner: Address;
+  borrowedAssetsRecipient: Address;
+}) {
+  return encodeFunctionData({
+    abi: IMorphoBlueFlashLoanAssetManager,
+    functionName: "init",
+    args: [args.owner, args.borrowedAssetsRecipient],
+  });
+}
+
+export function deployProxy(args: {
```

Comment:
this looks like a general-use helper function that will be used for other dispatcher-owned beacon factories, should we move it to a separate file?